### PR TITLE
PV protection description

### PR DESCRIPTION
**Pull request by NickrenREN**
Documentation for PR: 
https://github.com/kubernetes/kubernetes/pull/58743
https://github.com/kubernetes/kubernetes/pull/59169

Target release: 1.10

/cc @pospispa @jsafrane @heckj

### CODE REVIEW

File: docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -70,18 +70,19 @@ Pods use claims as volumes. The cluster inspects the claim to find the bound vol
...
 {: .note}
 
-When the [Storage Protection beta feature](/docs/tasks/administer-cluster/pvc-protection/) is enabled, if a user deletes a PVC in active use by a pod, the PVC is not removed immediately. PVC removal is postponed until the PVC is no longer actively used by any pods.
+When the [Storage Object in Use Protection beta feature](/docs/tasks/administer-cluster/storage-object-in-use-protection/) is enabled, if a user deletes a PVC in active use by a pod, the PVC is not removed immediately. PVC removal is postponed until the PVC is no longer actively used by any pods, and also if admin deletes a PV that is bound to a PVC, the PV is not removed immediately. PV removal is postponed until the PV is not bound to a PVC any more.
 
 You can see that a PVC is protected when the PVC's status is `Terminating` and the `Finalizers` list includes `kubernetes.io/pvc-protection`:
+
 ```shell
-kubectl describe pvc hostpath
+$ kubectl describe pvc hostpath
```
Comment (heckj) [inline docs/concepts/storage/persistent-volumes.md]: please remove the `$` from the example - per https://kubernetes.io/docs/home/contribute/style-guide/#dont-include-the-command-prompt

File: docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -94,6 +95,28 @@ Finalizers:    [kubernetes.io/pvc-protection]
 ...
 ```
 
+You can see that a PV is protected when the PV's status is `Terminating` and the `Finalizers` list includes `kubernetes.io/pv-protection` too:
+
+```shell
+$ kubectl describe pv task-pv-volume
```
Comment (heckj) [inline docs/concepts/storage/persistent-volumes.md]: please remove the `$` - per https://kubernetes.io/docs/home/contribute/style-guide/#dont-include-the-command-prompt

File: docs/tasks/administer-cluster/storage-object-in-use-protection.md [hunk 3]
```diff
@@ -215,6 +216,95 @@ Warning  FailedScheduling  18s (x4 over 21s)  default-scheduler  persistentvolum
...
+
+```shell
+NAME             CAPACITY     ACCESS MODES   RECLAIM POLICY   STATUS        CLAIM                   STORAGECLASS   REASON    AGE
+task-pv-volume   1Gi          RWO            Delete           Terminating   default/task-pv-claim   standard                 59s
+
+```
+-  Delete the PVC and verify that the PV is removed too.
+
+```shell
+$ kubectl delete pvc task-pv-claim
```
Comment (heckj) [inline docs/tasks/administer-cluster/storage-object-in-use-protection.md]: please remove the `$` - https://kubernetes.io/docs/home/contribute/style-guide/#dont-include-the-command-prompt

**Comment by heckj** [comment]
:wave: @pospispa @jsafrane - I would appreciate it if one of you would be kind enough to review this PR for technical accuracy. If it's good, please put an `/lgtm` comment on the PR, and we'll take that as an indicator of correctness from you.

File: docs/concepts/storage/persistent-volumes.md (hunk 1 above)
Comment (NickrenREN) [reply docs/concepts/storage/persistent-volumes.md]: done, thanks

File: docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (NickrenREN) [reply docs/concepts/storage/persistent-volumes.md]: done

File: docs/tasks/administer-cluster/storage-object-in-use-protection.md (hunk 3 above)
Comment (NickrenREN) [reply docs/tasks/administer-cluster/storage-object-in-use-protection.md]: done

File: _data/tasks.yml [hunk 4]
```diff
@@ -171,6 +171,7 @@ toc:
   - docs/tasks/administer-cluster/ip-masq-agent.md
   - docs/tasks/administer-cluster/dns-custom-nameservers.md
   - docs/tasks/administer-cluster/pvc-protection.md
```
Comment (pospispa) [inline _data/tasks.yml]: `pvc-protection.md` file was renamed. Could it be deleted from `tasks.yml`?

Comment (NickrenREN) [reply _data/tasks.yml]: oh, i forgot it. done, thanks

**Pull request merged**